client-src/elements/chromedash-admin-blink-page.ts

```diff
@@ -191,8 +186,8 @@ export class ChromedashAdminBlinkPage extends LitElement {
     if (location.hash) {
       const hash = decodeURIComponent(location.hash);
       if (hash) {
-        const el = this.shadowRoot.querySelector(hash);
-        el.scrollIntoView(true, {behavior: 'smooth'});
+        const el = this.shadowRoot!.querySelector(hash);
+        el!.scrollIntoView({behavior: 'smooth'});
       }
     }
   }
```

Comment:
You missed a use of `<chromedash-admin-blink-component-listing .id=...` in the section below this that won't match up with your renamed `.componentId` property.

Reply:
good catch. thanks